docs/readmes/lte/suci_extensions.md

```diff
@@ -0,0 +1,92 @@
+---
+id: suci_extensions
+title: SUCI Extensions
+hide_title: true
+---
+
+# SUCI Extensions
+
+SUCI (Subscription Concealed Identifier) is a privacy-preserving identifier containing the concealed SUPI (Subscription Permanent Identifier). The home network public key identifier encrypts SUPI and allows UE to authenticate with the network. The SUCI format is shown below.
+
+![SUCI Format](assets/lte/suci_format.png?raw=true "SUCI Format")
+
+The home network public key identifier(s) for the private/public key pairs used for subscriber privacy shall be stored in the subscriberdb of the Access Gateway (AGW). Subscriberdb should include a Subscription Identification De-concealing Function (SIDF), which contains an algorithm for decrypting SUCI using the home network private key.
+
+This document gives an overview of the SUCI Extensions functionality present in Magma, its architecture, configuration, feature matrix.
+
+## Use Case
+
+The UE generates and transmits the SUCI using a protection scheme, i.e., as one of the Elliptic Curve Integrated Encryption Scheme (ECIES) profiles, and the public key that was securely provisioned in the home network's control.
+
+The encryption and decryption mechanism specified in TS 33.501 is shown below.
+
+- According to 3GPP TS 33.501, the UE generates SUCI with the public key of the home network using an ECIES-based protection scheme.
+- According to 3GPP TS 33.501 clause C.3.2 (Processing on the UE side), the UE shall use the home network's provisioned public key and construct a new ECC (Elliptic Curve Cryptography) ephemeral public/private key pair based on the ECIES parameters provisioned by the home network.
+- According to 3GPP TS 33.501 clause C.3.3 (Processing on the home network side), the home network must use the UE's received ECC ephemeral public key and the home network's private key to de-conceal an SUCI.
+
+## Architecture
+
+Subscriberdb of AGW shall periodically fetch the list of SuciProfiles from Orc8r. Based on the home network public key identifier received in the authentication request, corresponding private key is retrieved and used to de-conceal the SUCI in the authentication request. Each public/private key pair is not subscriber specific, and can be shared among multiple subscribers.

```

Comment:
Consistent naming of `SUCI profiles`. Also further below.

Reply:
Addressed Review Comment